### ISSUE TITLE
Create URL path for directly signing user in

### ISSUE THREAD

**Issue by michaelsena** [3box/3box-dapp#416]
Something like https://3box.io/hub/login

** If hit this path, dump chosenProvider from localstorage, and display web3connect selection modal.

If users are sent to that path, they would be directly signed in including being shown the modals. They would not need to click a button to trigger the login process.

This was a request from dGaming, since they're sending people to 3box.io/hub/create to create profiles. They want as little friction as possible.

**Comment by michaelsena** [comment]
@oznekenzo how would we think about doing this with web3connect provider? Just having the user land on a page with the web3connect modal already open so they can just select their provider and then be signed in.

The goal is to direct users to a more specific route than just 3box.io/hub, where they still need to click a button. Instead we want to direct them straight away to 3box.io/hub/login or something where the web3connect modal is already on the screen. This would help with conversion when partners are directing their users to our site for them to create or modify their profile.

**Comment by oznekenzo** [comment]
@michaelsena you can basically already do what dGaming is requesting so long as you send the user to 3box.io/[ethaddress]/[protectedRoute], protectedRoute being one of [activity, details, collectibles, data, edit, following]

As for doing this with web3connect provider - exactly what you said.   Implementing it into our currently model is a little awkward because its showing a somewhat logged in interface with the web3connect modal on top - when you haven't logged in yet.  So I added within the web3connect implementation issue to create another intermediate webpage that will just be a prompt to pick your provide / give you the existing web3connect modal.  

TLDR: most of this will be done by the end of this epic - though creating the new route for it might not be
